frontend/src/pages/Profile.jsx

```diff
@@ -61,26 +69,48 @@ class Profile extends Component {
         )
     }
 
+    componentDidMount() {
+        this.getProfile();
+    }
+
+    getProfile() {
+        var urlPath = "/api/author/"
+        var authorId = store.getState().loginReducers.userId.split("thor/");
+        const path = urlPath + authorId[1], requireAuth = true;
+        HTTPFetchUtil.getRequest(path, requireAuth)
+        .then((httpResponse) => {
+            if (httpResponse.status === 200) {
```

Comment:
I think you should have an else block here to catch for the case of the status being 400 or something not 200.

Reply:
It also might be good to maybe build or use  a component to display an error message in the case we get a 400 or something.

Reply:
your wish is my command